api/api-doc/storage_service.json

```diff
@@ -833,6 +833,35 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/repair_status/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Query the repair status and return when the repair is finished",
+               "type":"string",
+               "enum":[
+                  "RUNNING",
```

Comment:
How can it ever return "RUNNING"?

However, I suggest to add a timeout parameter (and if it times out, return RUNNING). This is so that the caller can detect a node failure (but failing if it did not get a response after timeout+slack)

Reply:
Because I have adding "timeout" parameter in mind: https://github.com/scylladb/scylla/pull/6743#issuecomment-652745063